examples/custom-prefetch-methods/custom_prefetch_methods.md

```diff
@@ -0,0 +1,21 @@
+# GraphQL service - Custom prefetch methods
+
+The `prefetchMethodName` field of the `graphql:ResourceConfig` annotation is used to override the default prefetch method name. This configuration is useful when the Query and Mutation operations have fields with the same name, and the user wants to use different prefetch methods for each field, or when the user wants to use a different name for some other reason. To override the default prefetch method name, annotate the target field with the `graphql:ResourceConfig` annotation and specify the prefetch method name as the value of the `prefetchMethodName` field.
```

Comment:
Let's start with what is a custom prefetch method. (A brief intro would be enough, this is just to make sure when someone opens this example first, they have some understanding about what's going on here).

Reply:
Updated with [a45f671](https://github.com/ballerina-platform/ballerina-distribution/pull/4776/commits/a45f67153e0d88f235e99706b0bdd050b20d7a0b)